app/javascript/packages/webauthn/enroll-webauthn-device.ts

```diff
@@ -109,11 +111,13 @@ async function enrollWebauthnDevice({
   const authenticatorDataFlagsValue = authenticatorData
     ? new Uint8Array(authenticatorData)[32]
     : undefined;
+  const aaguid = arrayBufferToBase16(authenticatorData.slice(37,37 + 16))
 
   return {
     webauthnId: arrayBufferToBase64(credential.rawId),
     attestationObject: arrayBufferToBase64(response.attestationObject),
     clientDataJSON: arrayBufferToBase64(response.clientDataJSON),
+    aaguid,
```

Comment:
do we have any JS unit tests for this we can use to verify the byte stuff works?